### DISCUSSION TITLE
Every query times out after updating to 3.8.1

### DISCUSSION THREAD

**Discussion by Mw3y** [prisma/prisma#11308] [Q&A]
Hey, I just updated prima 2.30.3 to 3.8.1 and I'm getting this error on each query:
```
  Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: ConnectionError(Timed out during query execution.) })
```
I'm wondering if I missed something in the upgrade guide. I'm using an sqlite database and nodejs v16.13.2. I have multiple prisma clients connected to the database.

**Comment by austincrim** [comment]
Hey @Mw3y 👋 ,

Sorry you are seeing this issue! Would you mind re-running your application with all the Prisma debug logs turned on? You can read how on [this docs page](https://www.prisma.io/docs/concepts/components/prisma-client/debugging).

The short version is you need to prepend `export DEBUG=*` before your start command.

**Reply by Mw3y** [reply]
It shows less timeout errors (and when it shows one the query still works now) and it looks like most of the queries are working "normally".

I have another error now: 
```
Error:
Invalid `this.prisma[this.sessionModelName].update()` invocation in
node_modules\@quixo3\prisma-session-store\dist\lib\prisma-session-store.js:534:81

   531 existingSession = _b.sent();
   532 if (!(existingSession !== null)) return [3 /*break*/, 5];
   533 existingSessionData = __assign(__assign({}, this.serializer.parse((_a = existingSession.data) !== null && _a !== void 0 ? _a : '{}')), { cookie: session.cookie });
→  534 return [4 /*yield*/, this.prisma[this.sessionModelName].update(
  Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: ConnectionError(Timed out during query execution.) })
    at cb (node_modules\@prisma\client\runtime\index.js:38695:17)
```

**Reply by pantharshit00** [reply]
Hello @Mw3y , our apologies for the delay in getting back to you! Due to the large number of questions in our community channels, unfortunately this slipped through the cracks. It's very important for us to address your question, so let us know in a comment if this is still an open discussion. 

I will recommend you to upgrade to latest version and try again if this is still an issue. Do let me know if you are still facing this.

**Reply by Mw3y** [reply]
Hello, unfortunately it is still an issue! Recently I tried upgrading my ubuntu server which caused prisma 2.x.x to stop working due to an openssl upgrade. When updating prisma to the latest version, the error was still there. A lot of queries were timing out.

**Reply by Mw3y** [reply]
I forgot to mention that I'm using sqlite.

**Comment by Jolg42** [comment]
Note: there are multiple issues about this now all grouped under https://github.com/prisma/prisma/issues?q=is%3Aissue+is%3Aopen+label%3A%22topic%3A+Timed+out+during+query+execution%22 One of them is https://github.com/prisma/prisma/issues/10403 which points out that our error message here is currently highly misleading and should be changed/fixed.

_Originally posted by @janpio in https://github.com/prisma/prisma/issues/11789#issuecomment-1242518381_

**Reply by Jolg42** [reply]
So in your case, it's probably that the database is busy/locked

**Reply by Mw3y** [reply]
> So in your case, it's probably that the database is busy/locked

What could be causing this issue? Everything works fine with prisma 2.x.x.

**Reply by pantharshit00** [reply]
I would say it would be best if you can report this in a form of bug report with proper reproduction. Right now this is getting lost time to time due to it being a discussion. 

If you have some time, can you please fill out this bug report form here: https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
